components/ParallaxScroll.js

```diff
@@ -0,0 +1,24 @@
+export default class Parallax extends React.Component {
+  componentDidMount () {
+    window.addEventListener('load', this.scrollLoop, false)
```

Comment:
Please add a listener to `scroll` event instead.

We only want to update the parallax effect on scroll only, not continuously.

```js
     window.addEventListener('load', this.updateParallax, false)
     updateParallax()
```